lib/cutensornet/src/libcutensornet.jl

```diff
@@ -484,8 +484,8 @@ end
     initialize_context()
     @ccall libcutensornet.cutensornetContractSlices(handle::cutensornetHandle_t,
                                                     plan::cutensornetContractionPlan_t,
-                                                    rawDataIn::Ptr{Ptr{Cvoid}},
-                                                    rawDataOut::Ptr{Cvoid},
+                                                    rawDataIn::Ptr{CuPtr{Cvoid}},
```

Comment:
You also have to edit this in `res/wrap/cutensornet.toml`

Reply:
right sorry, will add

Reply:
This is a recent change, you couldn't have known 🙂 Should probably add a "this is autogenerated" header, or something.